src/envs.js

```diff
@@ -0,0 +1,21 @@
+module.exports = () => {
+  const ENVS = process.env.ENVS;
+
+  if (!ENVS) return '';
+
+  let envs;
+  try {
+    envs = JSON.parse(ENVS);
+  } catch (error) {
+    return '';
```

Comment:
Should we log when this happens for debugging? I think we should let this exception bubble up and cause the deployment to fail.